packages/common/src/ui/components/buttons/standard/DialogButton.tsx

```diff
@@ -44,12 +45,14 @@ export const DialogButton: React.FC<DialogButtonProps> = ({
   onClick,
   variant,
   disabled = false,
+  autoFocus = false,
 }) => {
   const t = useTranslation('common');
   const { variant: buttonVariant, icon, labelKey } = getButtonProps(variant);
 
   return (
     <ButtonWithIcon
+      autoFocus={autoFocus}
```

Comment:
off road wanted the alert modal to auto focus the `OK` so I can `enter` easier 🤷